### PR TITLE
Added API for getting users based on selected institution

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-6656](https://openedx.atlassian.net/browse/ECOM-6656)

Api can be accessed at `publisher/api/admins/organization/:id/`
 
- [x] Added api for getting organization group users
- [x] Added tests for api

### CODE REVIEW

File: course_discovery/apps/publisher/api/views.py [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
     queryset = CourseUserRole.objects.all()
     serializer_class = CourseUserRoleSerializer
+
+
+class OrganizationGroupUserView(ListAPIView):
+    serializer_class = UserSerializer
+    permission_classes = (IsAuthenticated,)
+
+    def get_queryset(self):
+        organization = get_object_or_404(OrganizationExtension, pk=self.kwargs.get('pk'))
```
Comment (awais786) [inline course_discovery/apps/publisher/api/views.py]: Here i think you will receive the organization id not the PK of OrganizationExtension.
Please double check.

File: course_discovery/apps/publisher/api/serializers.py [hunk 2]
```diff
@@ -21,3 +22,11 @@ def validate(self, data):
             validated_values.update({'changed_by': request.user})
 
         return validated_values
+
+
+class UserSerializer(serializers.ModelSerializer):
```
Comment (awais786) [inline course_discovery/apps/publisher/api/serializers.py]: Name should be some thing like GroupUserSerializer

File: course_discovery/apps/publisher/api/views.py (hunk 1 above)
Comment (awais786) [inline course_discovery/apps/publisher/api/views.py]: It should be organization id not the pk of the OrganizationExtension.

**Comment by tasawernawaz** [comment]
@awais786 Feedback addressed, please review again.

**Review by awais786 — APPROVED** [review]

**Comment by tasawernawaz** [comment]
@bderusha Please review this.

File: course_discovery/apps/publisher/api/urls.py [hunk 3]
```diff
@@ -1,10 +1,9 @@
...
 
-from course_discovery.apps.publisher.api.views import CourseRoleAssignmentView
+from course_discovery.apps.publisher.api.views import CourseRoleAssignmentView, OrganizationGroupUserView
 
 urlpatterns = [
-    url(
-        r'^course_role_assignments/(?P<pk>\d+)/$', CourseRoleAssignmentView.as_view(), name='course_role_assignments'
-    ),
+    url(r'^course_role_assignments/(?P<pk>\d+)/$', CourseRoleAssignmentView.as_view(), name='course_role_assignments'),
+    url(r'^admins/organization/(?P<pk>\d+)/$', OrganizationGroupUserView.as_view(), name='organization_group_users'),
```
Comment (bderusha) [inline course_discovery/apps/publisher/api/urls.py]: `r'^admins/organizations/(?P<pk>\d+)/users$'`

**Review by bderusha — APPROVED** [review]
One API endpoint name change then :shipit:

**Pull request merged**